File: tests/fixtures/code_samples/java/sample_026.java
// Sample 26: small utility.
package samples;

import java.util.List;

public final class Sample026 {
    private Sample026() {}

    public static int operation(List<Integer> xs) {
        int total = 26;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 26) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_026.py
"""Sample 26: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_26(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 26."""
    total = 26
    for x in xs:
        total += int(x)
    return total


def operation_26_pure(value: int) -> int:
    return (value * 26) % 7919

